chia/plotting/manager.py

```diff
@@ -272,8 +272,14 @@ def process_file(file_path: Path) -> Optional[PlotInfo]:
                 except Exception as e:
                     log.error(f"Failed to open file {file_path}. {e}")
                     return None
-                if stat_info.st_mtime == self.plots[file_path].time_modified:
-                    return self.plots[file_path]
+                if stat_info.st_mtime != self.plots[file_path].time_modified:
+                    log.info(
+                        f"Ignoring modification time mismatch for {file_path}: "
+                        f"({stat_info.st_mtime}) vs ({self.plots[file_path].time_modified})"
+                    )
+
+                return self.plots[file_path]
```

Comment:
so if `file_path` is found in `self.plots` we return that plot unconditionally.
The way this works today, it looks like we load the plot again, if it has an updated timestamp. But presumably if the plot is the same as before, it will just replace the previous entry, right? it won't cause a duplicate.

Reply:
It's worse, the code today in this case eventually falls into another if condition where this function essentially errors out and returns None - and this plot is then more or less lost and doesn't come back. 
That said, this fix was somewhat speculative and I hope to hear back whether it is an actual problem.
 I think it's unusual to have the stat call return something different here, but I don't think the code intended to throw away the plot in this case.

Reply:
I see. I seem to recall unsophisticated filesystems (such as FAT) do weird things with file timestamps sometimes, say, around daylight savings switch-overs.

Reply:
Agreed, Adam and I talked and we both thought, given all the different filesystems and implementations, and NFS/CIFS/USB, etc, that this check was probably not useful and we could just remove it. I decided to add in the log as an interim step

Reply:
In talking with Kevin, we decided to remove this check entirely